tests/test_utils.py

```diff
@@ -0,0 +1,16 @@
+# IMPORTS Astro:
+from astropy.io import fits
+from astropy.table import Table
+
+# IMPORTS Internal:
+from phrosty.utils import get_mjd
+
+def test_get_mjd():
+    pointing = 0
+    test_path = 'testdata/Roman_TDS_obseq_11_6_23.fits'
```

Comment:
Generalize this to refer to the directory of the file being tested, rather than from where the test is being run.  I.e., `pytest tests/test_utils.py` should work.  

`test_path = os.path.join(os.path.dirname(__file__), "testdata", "Roman_TDS_obseq_11_6_23.fits")